Package/Windows/Product.wxs

```diff
@@ -2,6 +2,11 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
      xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
 
+  <?if $(var.InstallerKind) = Proprietary ?>
+    <?define AndroidSupport ?>
+    <?define iOSSupport ?>
```

Comment:
iOS is not proprietary. Is the goal here just to allow building the installer without having a Mac build host set up?

A comment might be helpful here in the future, if that's the case. Or maybe we could decide whether or not to include iOS support separately, so that somebody with proprietary bits could build an installer _with_ Android and _without_ iOS?

Reply:
Oh, hmm. Good point. I can split that out into a separate property I guess and check explicitly for the app having been built.

Reply:
But yes - the goal is to allow the installer to build without having built some components that otherwise will build as part of our official VSTS builds.